### DISCUSSION TITLE
HTML Overridden by Express JS

### DISCUSSION THREAD

**Discussion by Great2BNate** [deepgram/community#69] [General help] [answered]
Hi, I followed the tutorial here, and it ended with using HTML Form without JS, and linked to a CODEPEN with JS. Thank you for that! However, in Replit, the HTML Form is getting overridden by the express JS. Therefore, I expected the tutorial to show us how to use Express JS to insert the HTML instead. Thanks!

**Comment by BekahHW** [comment]
Can you provide a link to the tutorial you used?

**Reply by Great2BNate** [reply]
Thank you for taking the time to help! 
I believe, if my memory serves me, that this is the tutorial I used: https://www.geeksforgeeks.org/create-a-music-player-using-javascript/

Here's a link to my replit: https://replit.com/@Great2BNate/HummDrum-Playlist#index.js

Again, thank you for offering assistance.

**Comment by BekahHW** [comment, marked as answer]
This resource may help:
- [https://www.digitalocean.com/community/tutorials/use-expressjs-to-deliver-html-files](https://www.digitalocean.com/community/tutorials/use-expressjs-to-deliver-html-files)

And you might be interested in this post:

- [Transcriptions Without a Server Using Netlify and Deepgram](https://blog.deepgram.com/transcription-netlify-functions)

**Reply by Great2BNate** [reply]
Thank you for the added resources. Very generous of you. I'll look into them.
